config/Configuration.go

```diff
@@ -15,23 +15,26 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	yamlFormat "github.com/ghodss/yaml"
+	"github.com/guzzlerio/corcel/core"
 	"github.com/imdario/mergo"
 	"github.com/mitchellh/go-homedir"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 //Configuration ...
 type Configuration struct {
-	Iterations int           `json:"iterations"`
-	Random     bool          `json:"random"`
-	Summary    bool          `json:"summary"`
-	LogLevel   log.Level     `json:"log-level"`
-	Workers    int           `json:"workers"`
-	Duration   time.Duration `json:"duration"`
-	WaitTime   time.Duration `json:"wait-time"`
-	Progress   string        `json:"progress"`
-	Plan       bool          `json:"plan"`
-	FilePath   string
+	Iterations     int
+	Random         bool
+	Summary        bool
+	SummaryFormat  string `json:"summary-format"`
+	SummaryBuilder core.SummaryBuilder
```

Comment:
I would say a SummaryBuilder takes a configuration as opposed to a Configuration being coupled to a SummaryBuilder.